activity.py

```diff
@@ -26,12 +29,15 @@ def __init__(self, handle):
 
         self.game = jump.SolitaireMain()
         self.build_toolbar()
-        self._pygamecanvas = sugargame.canvas.PygameCanvas(self)
+
+        self.game.canvas = self._pygamecanvas = \
+            sugargame.canvas.PygameCanvas(self,
+                main=self.game.SuperLooper,
+                modules=[pygame.display, pygame.font])
```

Comment:
I've not checked the rest of the code that you haven't changed; but you did change a button that mutes sound, so if the sound uses `pygame.mixer` you might add that module to the list of modules at this line.